src/eve/traits.py

```diff
@@ -68,10 +68,6 @@ def _collect_symbols_validator(  # type: ignore  # validators are classmethods
         values["symtable_"] = cls._collect_symbols(values)
         return values
 
-    def collect_symbols(self) -> None:
-        self.symtable_ = dict()
-        self.symtable_ = self._collect_symbols(self)
-
```

Comment:
FYI, this was just unused duplicated logic from elsewhere.

Reply:
Just a clarification here: this is not necessarily meant to be used, it's just an additional function that the trait adds to the node class, in case the user wants to explicitly trigger a re-collection of symbols. It can be deleted if you think it's useless though